src/Configuration.h

```diff
@@ -50,7 +50,8 @@ struct QMCTraits
 {
   enum
   {
-    DIM = OHMMS_DIM
+    DIM = OHMMS_DIM,
+    DIM_VGL = OHMMS_DIM + 2
```

Comment:
It might be useful to have a comment to explain what VGL means and where the +2 comes from.